client/src/i18n/en/inventory.json

```diff
@@ -41,8 +41,8 @@
         "INVENTORY_PRICE_INFO" : "inventory_price : the sale price of the inventory in the enterprise currency",
         "COLUMNS_INFO" : "Note about some columns",
         "SELLABLE" : "Sellable",
-        "VIEW_ARTICLES_IN_STOCK" : "View Items in Stock",
-        "VIEW_ITEM_IN_INVENTORY" : "View Item in Inventory",
+        "VIEW_ARTICLES_IN_STOCK" : "View Articles in Stock",
+        "VIEW_ITEM_IN_INVENTORY" : "View Article in Inventory",
```

Comment:
Can you even update this translation key?  "VIEW_ITEM" -> "VIEW_ARTICLE"?

Reply:
Done